omigami/data/__init__.py

```diff
@@ -0,0 +1,21 @@
+from omigami.data.data_types import (
```

Comment:
Sorry for being picky but I dislike the name data for a module.

Reply:
Don't you think types should be also separate?

Reply:
I much preferred data_structures or just data_models or entities.

Reply:
>Don't you think types should be also separate?
you mean `omigami.types`? the name `types` conflicts with other package file and was resulting in error in tests.